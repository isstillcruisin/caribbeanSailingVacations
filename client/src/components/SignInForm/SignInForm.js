import React from "react";

const SignInForm = props => (
  <form>
    <div className="form-group">
      <label htmlFor="email">
        <strong>Email</strong>
      </label>
      <input
        className="form-control"
        id="email"
        type="text"
        value={props.email}
        placeholder="Email"
        name="email"
        onChange={props.handleInputChange}
        required
      />
      <label htmlFor="password">
        <strong>Password</strong>
      </label>
      <input
        className="form-control"
        id="password"
        type="password"
        value={props.password}
        placeholder="Password"
        name="password"
        onChange={props.handleInputChange}
        required
      />
    </div>
    <div className="pull-right">
      <button
        onClick={props.handleSignIn}
        type="submit"
        className="btn btn-lg btn-danger"
      >
        Submit
      </button>
    </div>
  </form>
);

export default SignInForm;
