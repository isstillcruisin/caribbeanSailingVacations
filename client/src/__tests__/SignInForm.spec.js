import React from "react";
import { create } from "react-test-renderer";
import SignInForm from '../components/SignInForm';

describe("SignInForm component", () => {
  test("Matches the snapshot", () => {
    const form = create(<SignInForm 
      email='test@testerson.com'
      password='aPassword'
    />);
    expect(form.toJSON()).toMatchSnapshot();
  });
});