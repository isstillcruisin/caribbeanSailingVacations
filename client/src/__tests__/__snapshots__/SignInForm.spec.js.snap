// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInForm component Matches the snapshot 1`] = `
<form>
  <div
    className="form-group"
  >
    <input
      className="form-control"
      id="email"
      name="email"
      placeholder="Email"
      required={true}
      type="text"
      value="test@testerson.com"
    />
    <input
      className="form-control"
      id="password"
      name="password"
      placeholder="Password"
      required={true}
      style={
        Object {
          "marginLeft": ".5rem",
        }
      }
      type="password"
      value="aPassword"
    />
  </div>
  <div
    className="pull-right"
  >
    <button
      className="btn btn-lg btn-danger"
      type="submit"
    >
      Submit
    </button>
  </div>
</form>
`;
